drivers/volume/portworx/portworx.go

```diff
@@ -1098,6 +1098,44 @@ func (d *portworx) GetClusterPairingInfo() (map[string]string, error) {
 	return pairInfo, nil
 }
 
+func (d *portworx) DecommissionNode(n node.Node) error {
+
+	if err := d.StopDriver([]node.Node{n}, false); err != nil {
```

Comment:
This alone is not the customer steps. See step 2: https://docs.portworx.com/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/uninstall/decommission-a-node/#2-decommission-portworx

You also need ` px/enabled=remove ` so that the daemonset pod goes away.